ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces_and_l3_edge/router_bgp.py

```diff
@@ -36,13 +36,21 @@ def router_bgp(self) -> dict | None:
 
             neighbor = {
                 "remote_as": p2p_link["data"]["peer_bgp_as"],
+                "peer": p2p_link["data"]["peer"],
                 "description": p2p_link["data"]["peer"],
                 "peer_group": self.shared_utils.bgp_peer_groups["ipv4_underlay_peers"]["name"],
             }
 
             # RFC5549
             if self.shared_utils.underlay_rfc5549:
-                neighbor_interfaces.append({"name": p2p_link["data"]["interface"], **neighbor})
+                neighbor_interfaces.append(
+                    {
+                        "name": p2p_link["data"]["interface"],
+                        "remote_as": neighbor["remote_as"],
+                        "description": neighbor["description"],
+                        "peer_group": neighbor["peer_group"],
+                    }
+                )
```

Comment:
Why are you changing this? For consistency we would also want to output `peer` for the `neighbor_interfaces` even if you don't use it in validate state (yet).

Reply:
You would have to add it to the schema obv.

Reply:
I was debating to add it or not since it will modify all the RFC5549 stuff.

Reply:
I am in favor of adding peer here - and running molecule ;)  You will need to do it in other places as well (mlag, ...)